lib/JMAP/Tester.pm

```diff
@@ -256,4 +283,38 @@ sub request {
   });
 }
 
+=method upload
+
+  my $blobject = $tester->upload($mime_type, $blob, \%arg);
```

Comment:
Maybe $blob should be $blob_ref or something to indicate it's a scalar ref.

\%arg looks like extra here that isn't actually used?